pkg/crc/ssh/ssh_test.go

```diff
@@ -88,6 +102,7 @@ func createSSHServer(t *testing.T, listener net.Listener, clientKey *ecdsa.Priva
 
 	go func() {
 		for {
+			fmt.Printf("#goroutines default channel: %d\n", runtime.NumGoroutine())
```

Comment:
Not sure this `Printf` and the similar one below is needed?

Reply:
Ah that was for debug purposes :) , will remove

Reply:
It still seems to be present?